src/main/java/Deadline.java

```diff
@@ -0,0 +1,61 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+<<<<<<< HEAD
+/**
+ * Supports the creation of Deadline objects.
+ */
+public class Deadline extends Task{
+=======
+public class Deadline extends Task {
+>>>>>>> branch-A-CodingStandard
```

Comment:
Perhaps you missed out a merge conflict here?

Reply:
I did, thanks for pointing it out!